### PR TITLE
Added scroll controls to timeline category display

### PR DESCRIPTION
**Pull request by engelhamer**
timeline-category-scroll

### CODE REVIEW

File: src/vue/src/components/category/CategoryDisplay.vue [hunk 1]
```diff
@@ -62,22 +76,50 @@ export default {
     computed: {
         infoModalID () { return `${this.id}-category-information` },
     },
+    methods: {
+        scrollLeft () {
+            this.$refs[`${this.id}-category-display`].scrollBy({
+                left: -this.$refs[`${this.id}-category-display`].clientWidth,
```
Comment (mjvkeulen) [inline src/vue/src/components/category/CategoryDisplay.vue:82]: [NIT]: Combine methods and parametrize left or right as + or -

**Review by mjvkeulen — APPROVED** [review]

File: src/vue/src/components/category/CategoryDisplay.vue [hunk 2]
```diff
@@ -18,6 +26,12 @@
                 $nextTick(() => { $bvModal.show(infoModalID) })
             "
         />
+        <icon
+            v-if="categories.length > 0"
+            class="category-right"
+            name="caret-right"
+            @click.native="scrollRight"
+        />
```
Comment (Larspolo) [inline src/vue/src/components/category/CategoryDisplay.vue:34]: Arrows are still visible even when there is nothing to scroll through, would like to see them hidden if there is no overflow

Comment (engelhamer) [reply src/vue/src/components/category/CategoryDisplay.vue:34]: Most naive solution would require `$refs` in computed prop, which is not desirable. Defering to issue for now.

File: src/vue/src/components/category/CategoryDisplay.vue (hunk 1 above)
Comment (engelhamer) [reply src/vue/src/components/category/CategoryDisplay.vue:82]: I like the explicit names a slight bit better (without a wrapper or sign)

**Pull request merged**